src/modules/auto_fill_region/actions-test.js

```diff
@@ -34,5 +34,20 @@ describe('Autofill Actions', () => {
       const {guesstimate: {input}} = fillFn(location, metrics)
       expect(input).to.equal(startInput)
     })
+
+    it('fills properly with no translatable metrics, but some non-constant', () => {
+      const location = {row: 1, column: 2}
+      const metrics = [
+        {
+          id: '2',
+          readableId: 'VL',
+          location: {row: 0, column: 0},
+        },
+      ]
+      const fillFn = actions.fillDynamic(startMetric, startGuesstimate, direction)
+      const {metric, guesstimate} = fillFn(location, metrics)
+      console.log(metric, guesstimate)
+      expect(!metric && !guesstimate).to.equal(true)
```

Comment:
Do you need the console.log?  
Also, the next line is confusing. Maybe something more like,
``expect(!!guesstimate).to.equal(false)``
``expect(!!metric).to.equal(false)``

Although to be honest I'm really not sure what is happening there.